packages/react-percy-webpack/src/resolve/__tests__/getExtension-tests.js

```diff
@@ -2,22 +2,22 @@ import getExtension from '../getExtension';
 
 jest.mock('interpret', () => ({
   extensions: {
-    '.babel.js': null,
+    '.buble.js': null,
     '.coffee': null,
     '.js': null,
   },
 }));
 
 it('returns multipart JS file extension when mapped by interpret', () => {
-  const extension = getExtension('webpack.config.babel.js');
+  const extension = getExtension('webpack.config.buble.js');
 
-  expect(extension).toEqual('.babel.js');
+  expect(extension).toEqual('.buble.js');
 });
 
-it('returns JS when multipart JS file extension not mapped by interpret', () => {
+it('returns Babel JS when multipart JS file extension not mapped by interpret', () => {
   const extension = getExtension('webpack.config.foo.js');
 
-  expect(extension).toEqual('.js');
+  expect(extension).toEqual('.babel.js');
```

Comment:
We're using a library called interpret to resolve non-standard JS webpack config files (e.g. ES6 modules, coffeescript, etc.). This is exactly what webpack itself does when loading a config file. However, they require a `.babel.js` file extension if you want to use ES6 imports or other syntax not natively supported by Node. In order to always run babel on webpack config files even if they don't end in `.babel.js`, we just need to treat all `.js` files as `.babel.js` files instead. This will trigger interpret to load the babel transpiler.